recipe/doc/end_user_run_guide.md

```diff
@@ -0,0 +1,121 @@
+
+```
+conda install -c conda-forge cuda cuda-version=12.4
+```
+
+Let's break down this command.
+We are requesting the installation of two metapackages here, `cuda` and `cuda-version`.
+The `cuda` metapackage pulls in all the components of the CUDA Toolkit (CTK) and is roughly equivalent to installing the CUDA Toolkit with traditional OS package managers like apt or yum on Linux.
+Similarly to such package managers, the separate components may also be installed independently.
+The [`cuda-version` metapackage](https://github.com/conda-forge/cuda-version-feedstock/blob/main/recipe/README.md) is used to select the version of CUDA to install.
+This metapackage is important because individual components of the CTK are typically versioned independently (the current versions may be found in the [release notes](https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html).
+The `cuda-version` metapackage provides a standard way to install the version of a specific CUDA component corresponding to a given version of the CTK.
+This way, you never have to specify a particular version of any package; you just specify the `cuda-version` that you want, then list packages you want installed and conda will take care of finding the right versions for you.
+The above command will install all components of CUDA from the latest patch release of CUDA 12.4.
+
+_Warning: there are contents of OS CUDA Toolkit installs that are not available in the conda-forge packages, including_:
+- Driver libraries, such as (but not limited to)
+    - `libcuda`
+    - `libnvidia-ml` library
+- Older versions of CTK contained these
+    - Documentation
+    - Samples (use to be)
+    - Demo suite (limited subset of samples)
+- GDS
+    - Missing file system components
+- fabricmanager
+- `libnvidia_nscq`
+- Imex
+- Nsight-systems
+
+### Installing Subsets of the CTK
+
+Rather than installing all of CUDA at once, users may instead install just the packages that they need. 
+For example, to install just libcublas or libcusparse one may run:
+```
+conda install -c conda-forge libcublas cuda-version=<CUDA version>
+```
+The best way to get a current listing is to run
+```
+conda install --dry-run -c conda-forge cuda cuda-version=<CUDA version>
+```
+Original build order: https://github.com/conda-forge/staged-recipes/issues/21382.
+
+### Metapackages
+
+Existing conda documentation: https://docs.nvidia.com/cuda/cuda-installation-guide-linux/#conda-installation
+
+For convenience, a number of additional metapackages are available:
+- `cuda-runtime`: All CUDA runtime libraries needed to run a CUDA application
+- `cuda-libraries`: All libraries required to run a CUDA application requiring libraries beyond the CUDA runtime (such as the CUDA math libraries) as well as packages needed to perform JIT compilation.
+- `cuda-visual-tools`: GUIs for visualizing and profiling such as Nsight Systems and Nsight Compute
```

Comment:
Didn't we just say that we don't ship Nsight Systems above? This seems contradictory.

Reply:
Fixed this. We include compute but not systems.